lib/CMakeLists.txt

```diff
@@ -70,27 +70,8 @@ find_package(LLNL-Units REQUIRED)
 # Generate files for free scipp API functions
 include(scipp-functions)
 
-find_program(CCACHE ccache)
-if(CCACHE)
-  set(CMAKE_CXX_COMPILER_LAUNCHER ${CCACHE})
-endif()
-
-if(MSVC)
-  set(CLCACHE_PATH
-      ""
-      CACHE PATH "Path to installed clcache executable directory"
-  )
-  if(EXISTS ${CLCACHE_PATH})
-    set(CMAKE_VS_GLOBALS "CLToolExe=clcache.exe" "CLToolPath=${CLCACHE_PATH}"
-                         "TrackFileAccess=false"
-    )
-    message(STATUS "Will attempt to use clcache")
-  else(EXISTS ${CLCACHE_PATH})
-    message(STATUS "CLCACHE_PATH not specified. Not enabled.")
-    unset(CMAKE_VS_GLOBALS) # Clear all vs globals
-  endif(EXISTS ${CLCACHE_PATH})
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /constexpr:steps2000000")
-endif(MSVC)
+find_program(CCACHE ccache REQUIRED)
```

Comment:
Really required? Our package builds (in `conda build`) do not use ccache, since it does not provide any benefit there.